tests/upgrades/test_errata.py

```diff
@@ -253,15 +235,14 @@ def test_post_scenario_errata_count_installation(self, pre_upgrade_data):
         # waiting for errata count to become 0, as profile uploading take some
         # amount of time
         wait_for(
-            lambda: self._errata_count(ak=activation_key) == 0,
+            lambda: host.read().content_facet_attributes['errata_counts']['total'] == 0,
             timeout=400,
             delay=2,
             logger=logger,
         )
-        host = entities.Host().search(query={'search': f'activation_key={activation_key}'})[0]
-        assert host.content_facet_attributes['errata_counts']['total'] == 0
-        for package in FAKE_9_YUM_UPDATED_PACKAGES:
-            install_or_update_package(client_hostname=client_container_id, package=package)
```

Comment:
Does anyone know why we install packages at the end of the post-test?

Reply:
I too agree here there is no benefit here to installing the package if there are no post checks for that. I feel that we are just validating the errata count to become 0, we could also check the package version being updated.